var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -33,51 +33,73 @@ class Subversion(Package):
     version('1.8.13',    '8065b3698d799507fb72dd7926ed32b6')
     version('1.9.3',     'a92bcfaec4e5038f82c74a7b5bbd2f46')
 
+    variant('python', default=False, description='Provide python bindings')
+    variant('perl', default=False, description='Provide Perl bindings')
+    # variant('ruby', default=False, description='Provide ruby bindings')
+
     depends_on('apr')
     depends_on('apr-util')
     depends_on('zlib')
     depends_on('sqlite')
     depends_on('serf')
+    depends_on('expat')
+    depends_on('swig@:2.99')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('pkg-config', type='build')
 
-    # Optional: We need swig if we want the Perl, Python or Ruby
-    # bindings.
-    # depends_on('swig')
-    # depends_on('python')
-    # depends_on('perl')
-    # depends_on('ruby')
+    # Optional: We need swig if we want the Perl, Python or Ruby bindings.
+    extends('python', when='+python')
+    depends_on('perl@5.8:', when='+perl')
```

Comment:
Please figure out whether the Python bindings work with Python2, Python3 or both; and add clauses accordingly.  If you need that, you should add a second line, eg: `depends_on('python@1.4:17.6')`

Also, does `when=` work on `extends('python')`?  It should, but I'm not sure if it does yet.